### PR TITLE
Fix missing Solana icon for buy with ramp

### PR DESCRIPTION
**Pull request by muliswilliam**
Resolves https://github.com/brave/brave-browser/issues/23013

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
Screenshot 2022-05-24 at 21 00 26

### CODE REVIEW

File: components/brave_wallet_ui/utils/asset-utils.ts [hunk 1]
```diff
@@ -20,3 +21,14 @@ export const isSelectedAssetInAssetOptions = (selectedAsset: BraveWallet.Blockch
...
+export const getRampAssetSymbol = (asset: BraveWallet.BlockchainToken) => {
+  if (asset.symbol.toUpperCase() === 'BAT') {
+    // BAT is the only token on Ethereum Mainnet with a prefix on Ramp.Network
+    return 'ETH_BAT'
+  }
+
+  const rampNetworkPrefix = getRampNetworkPrefix(asset.chainId)
+  return rampNetworkPrefix !== '' ? `${rampNetworkPrefix}_${asset.symbol.toUpperCase()}` : asset.symbol
+
+}
```
Comment (Douglashdaniel) [inline components/brave_wallet_ui/utils/asset-utils.ts:34]: Needs to end with a new line.

File: components/brave_wallet_ui/utils/string-utils.ts [hunk 2]
```diff
@@ -10,3 +12,20 @@ export const isRemoteImageURL = (url?: string) =>
...
+    case BraveWallet.BINANCE_SMART_CHAIN_MAINNET_CHAIN_ID:
+      return 'BSC'
+    case BraveWallet.POLYGON_MAINNET_CHAIN_ID:
+      return 'MATIC'
+    case BraveWallet.SOLANA_MAINNET:
+      return 'SOLANA'
+    default:
+      return ''
+  }
+}
```
Comment (Douglashdaniel) [inline components/brave_wallet_ui/utils/string-utils.ts:31]: Same here!

**Review by Douglashdaniel — APPROVED** [review]
Looks good!
Just need to fix some lint issues and missing lines.

**Comment by srirambv** [comment]
Verification passed on 

Brave | 1.41.20 Chromium: 102.0.5005.61 (Official Build) nightly (64-bit)
-- | --
Revision | `0e59bcc00cc4985ce39ad31c150065f159d95ad3-refs/branch-heads/5005@{#819}`
OS | Windows 11 Version 22H2 (Build 22621.1)

- Verified Solana token icon is shown for `Buy with Ramp`
- Verified Solana token icon is shown when selecting token list

https://user-images.githubusercontent.com/17010094/170915972-961bb876-4367-4126-8916-a34cbd78ad2e.mov

**Pull request merged**